### ISSUE TITLE
[ISSUE] Carnet desktop for Linux doesnt connect to NextCloud server.

### ISSUE THREAD

**Issue by hardwareadictos** [CarnetApp/CarnetElectron#43]
Hello, good afternoon.

First of all i want to say how grate this app is, and big thanks to the development team who made this project possible.

I have a little issue when i try to connect Carnet to to my NextCloud server:

![imagen](https://user-images.githubusercontent.com/20403906/82576071-9fad9700-9b89-11ea-9adf-56ff619a47ea.png)

I dont have any issue with the Android app, it allows me to connect and sync my notes pretty fine with my phone, in fact, android app uses a more suitable workaround, as it allows you to browse inside your server to find the desired folder to sync notes (just a request a part from the issue report, could be posible to implement this behaviour on the Electron App?).

Tried different destination folders (including the default one), tried WebDAV address instead regular one (just to try luck) but nothing seems to work.

Let me know if you need any information please.

Thanks in advance.

Best regards.

**Comment by PhieF** [comment]
Hi,

do you have something like double authentication ?

thanks for your report

**Comment by hardwareadictos** [comment]
Hello! 

No, i dont have it. Its just a normal nextcloud home server, it does even have self-signed certificates.

Weird thing is that works just fine on the android app.

**Comment by hardwareadictos** [comment]
Hey, 

It's been weeks since there's not activity on this issue, could be related to self signed certificates?

Is it possible to integrate NextCloud browse sync settings Android App like?

Thanks in advance :)

Cheers.

**Comment by eandersons** [comment]
@hardwareadictos What URL are you using to connect to Nextcloud in the Carnet app? In macOS Carnet app I used https://nextcloud.my.domain/remote.php/webdav to connect to my Nextcloud server.

**Comment by hardwareadictos** [comment]
> @hardwareadictos What URL are you using to connect to Nextcloud in the Carnet app? In macOS Carnet app I used https://nextcloud.my.domain/remote.php/webdav to connect to my Nextcloud server.

Yes, i add the URL that my nextcloud server settings points me wich is the same you posted but adapted to my server name (it also works with the IP).

It's weird because it's not even processing the request, the error pops up instantly when pressing the connect button

**Comment by hardwareadictos** [comment]
I just migrated my server to another platform and i confirm that this issue persists

**Comment by hardwareadictos** [comment]
Hi, here is another screenshot from the issue:

![image](https://user-images.githubusercontent.com/20403906/85425175-2551a780-b579-11ea-8237-22e39a6c84e7.png)

Please, can somene guide me to workaround this? I want to is this app with my NextCloud Server, it has some potential. But the issue its been opened for more than a month without any support from the dev team.

Thanks in advance.

**Comment by Teuwu** [comment]
I do believe that this issue is related to self-signed certs, as I have 2 urls pointing to my nextcloud, 1 is with letsencrypt certs, and the second is with my self-signed cert, I have tried unsuccessfully to get carnet to accept the self signed certs.

**Comment by herrsei123** [comment]
I'm also having the same issue. Its also a server with self-signed certificate. Any fixes for this?

**Comment by EdoSecco** [comment]
I'm unable to sync on pc with Nextcloud WebDAV, too.
I have an account on Wölkli Nextcloud; while on the phone I managed to set the synchronization of Carnet via associating with Nextcloud app without any problem, there is no way to set on my pc (Ubuntu 20.04), the app is last version, taken from Snap; like the opener I inserted the same URL/user/pw but always get that error message.
I have other things on the pc synced with that Nextcloud instance so I don't think it's caused by this.

**Comment by loxK** [comment]
I have the same issue. It won't connect. I tried:
- https://[nextcloudhost]/remote.php/dav
- https://[nextcloudhost]/remote.php/webdav
- https://[nextcloudhost]

I have double authentication, so I generated an app password, I am using valid let's encrypt certificate.

**Comment by mementomori74** [comment]
im having the same problem. the android app sync just fine. The linux desktop version gives this error
![carnet](https://user-images.githubusercontent.com/49150534/148063763-1c0e4548-0b78-4cef-b8de-cff446b6128d.png)

**Comment by paulsybrandy** [comment]
So I have the same issue but what I did was I have the Nextcloud app syncing my my machine so I directed the carnet app to use the notes in that same folder but on my local machine. So the "/home/USER/nextcloud/documents/QuickNote/" which then the Nextcloud sync app syncs and then the Android app sees the changes. Basically I am making the Nextcloud sync app do the work of syncing that the Carnet should be doing.
